widgets/stats/index.js

```diff
@@ -3,12 +3,16 @@ var React = require('react');
 var utils = require('../../lib/utils.js');
 var defaultTemplate = require('./template.html');
 
-function stats({container = null, template = defaultTemplate}) {
+function stats({
+    container = null,
+    template = defaultTemplate,
+    transformData = null
+  }) {
   var Stats = require('../../components/Stats');
   var containerNode = utils.getContainerNode(container);
 
   if (container === null) {
-    throw new Error('Usage: stats({container})');
+    throw new Error('Usage: stats({container, template, transformData})');
```

Comment:
Should be `throw new Error('Usage: stats({container/*, template, transformData*/})');`